DESCRIPTION

```diff
@@ -15,6 +15,8 @@ Authors@R:
              role = c("aut"),
              email = "christoph@cynkra.com")
     )
+Author: Nicolas Bennett [aut, cre], David Granjon [aut], Christoph Sax [aut]
```

Comment:
Not sure here, we already have an Authors field with roles, no?

Reply:
Yes, I know, a mistake